examples-gallery/plot_sliding_block.py

```diff
@@ -0,0 +1,333 @@
+# %%
+""""
+Sliding a Block on a Road
+=========================
+A block, modeled as a particle is sliding on a road to cross a
+hill. The block is subject to gravity and speed dependent
+friction.
+Gravity points in the negative Y direction.
+A force tangential to the road is applied to the block.
+Three objective functions to me minimized may be selected:
+
+- selektion = 0: time to reach the end point is minimized
+- selektion = 1: time to reach the end point and the energy are minimized.
+- selektion = 2: energy consumed is minimized.
```

Comment:
Only two animations are displayed, shouldn't there be a result per selection?

Reply:
I only picked two of the three, to speed up the running time. Should I go back to all three?

Reply:
I'd remove option 1, as 0 and 2 are sufficient for an example and option 1 is totally dependent on how you weight the two components to the objective function.

Reply:
I found this 'relative weight' quite interesting mathematically - but I agree, it distracts from the example.
I will make the corrections this evening and make a new pull request.
It did come in a branch like it should?